README.markdown

```diff
@@ -138,7 +138,10 @@ It accepts a `opts` table argument. The following options are supported:
 * `no_random`
 
 	a boolean flag controls whether to randomly pick the nameserver to query first, if `true` will always start with the first nameserver listed. Defaults to `false`.
+* `doh`
```

Comment:
This field is confusing and can be split into two fields: `doh` is a boolean and `doh_method` is a string

Reply:
According to your previous comment with bootstrap DNS, maybe udp/tcp DNS list should be treated like bootstrap DNS servers. DoH list should be separate, `doh` as a boolean whether to enable DoH and `doh_method`.

Reply:
As a complete DoH client should provide such options, and reusing the original DNS list as bootstraping DNS would be confusing for users.